### DISCUSSION TITLE
WG Meeting Queries

### DISCUSSION THREAD

**Discussion by naveen-rn** [openshmem-org/specification#494] [Polls]
As we are starting back our WG meetings, we are seeing sparse attendance for the last few weeks. We are not sure on the reason behind this - is it because the regular attendees are not receiving meeting updates, or are there conflicts in current meeting timings. Please respond to the following poll.

**Comment by jdinan** [comment]
Was the gmail issue with the OpenSHMEM mailing list fixed?

**Reply by naveen-rn** [reply]
@tonycurtis can provide some updates - AFAIU - I'm not seeing emails on my gmail account. Looks like it is still not fixed.

**Comment by tonycurtis** [comment]
I think this is easy to solve, but requires going through OSSS/Steve.

**Comment by naveen-rn** [comment]
Discussion is closed as stale
